### ISSUE TITLE
APIMAN 2.2.0 in case of Big size Payload APIMAN Response time is very high 

### ISSUE THREAD

**Issue by shachindrasingh** [apiman/apiman#1921]
we are using APIMAN 2.2.0 with open JDK "openjdk version "1.8.0_222"" .

if we pass large Payload in any request apiman response time got increase. we check on backend module in that will return the response in mil second. but we got the response after 2 or 3 second.  that increase if we use the large payload.

we are using in Gateway tomcat apache-tomcat-9.0.58. 
if any configuration if required please let me know.

**Comment by msavy** [comment]
Please provide the information from the bug template, especially the JVM settings you are using such as `Xmx`.

Have you tried increasing `Xmx`?

**Comment by shachindrasingh** [comment]
Please find the JVM setting -Xms2g -Xmx2g

**Comment by msavy** [comment]
Which policies are you using?

**Comment by shachindrasingh** [comment]
Policy is our customize but we remove the policy and check the response time,  not much improvement,

**Comment by msavy** [comment]
Is it the same if you use the Wildfly or Vert.x Gateway?

**Comment by shachindrasingh** [comment]
Not yet check because we are using tomcat gateway  ..

**Comment by msavy** [comment]
OK, hopefully a community member also using the TomCat gateway will provide some input.

**Comment by wbeck-fr** [comment]
Is the option "Enable stateful request payload inspection" is disable ?

**Comment by shachindrasingh** [comment]
yes that is disable.. but what i observer that while apiman is sending the request to module endpoint at that time . system is taking the time.

**Comment by shachindrasingh** [comment]
any update on that
